larray/core/axis.py

```diff
@@ -998,6 +998,45 @@ def difference(self, other):
         seen = set(other)
         return Axis([l for l in self.labels if l not in seen], self.name)
 
+    def align(self, other, join='outer'):
+        """Align axis with other object using specified join method.
+
+        Parameters
+        ----------
+        other : Axis or label sequence
+        join : {'outer', 'inner', 'left', 'right'}, optional
+            Defaults to 'outer'.
+
+        Returns
+        -------
+        Axis
+            Aligned axis
+
```

Comment:
This docstring should be only read by developers or also users? If developers only, no need for section "See Also".